datasets/elkarhizketak/README.md

```diff
@@ -0,0 +1,102 @@
+ - train: 1,306 questions / 301 dialogues
+ - development:  161 questions / 38 dialogues
+ - test: 167 questions / 38 dialogues
+
+## Dataset Creation
+
+### Curation Rationale
+
+This is the first non-English conversational QA dataset and the first conversational dataset for Basque. Its small size presents a realistic low-resource scenario for conversational QA systems.
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+First we selected sections of Wikipedia articles about people, as less specialized knowledge is required to converse about people than other categories. In order to retrieve articles we selected the following categories in Basque Wikipedia: Biografia (Biography is the equivalent category in English Wikipedia), Biografiak (People) and Gizabanako biziak (Living people). We applied this category filter and downloaded the articles using a querying tool provided by the Wikimedia foundation. Once we retrieved the articles, we selected sections from them that contained between 175 and 300 words. These filters and threshold were set after some pilot studies where we check the adequacy of the people involved in the selected articles and the length of the passages in order to have enough but not to much information to hold a conversation.
+
+Then, dialogues were collected during some online sessions that we arranged with Basque speaking volunteers. The dialogues involve two crowd workers: (1) a student ask questions after reading a small introduction about the person, but without seeing the section text; and (2) a teacher answers the questions selecting a span of text of the section.
+
+#### Who are the source language producers?
+
+The language producers are Basque speaking volunteers which hold a conversation using a text-based chat interface developed for those purposes.
+
+## Additional Information
+
+### Dataset Curators
+
+The dataset was created by Arantxa Otegi, Jon Ander Campos, Aitor Soroa and Eneko Agirre from the [HiTZ Basque Center for Language Technologies](https://www.hitz.eus/) and [Ixa NLP Group](https://www.ixa.eus/) at the University of the Basque Country (UPV/EHU).
+
+### Licensing Information
+
+Copyright (C) by Ixa Taldea, University of the Basque Country UPV/EHU.
+
+This dataset is licensed under the Creative Commons Attribution-ShareAlike 4.0 International Public License (CC BY-SA 4.0).
+To view a copy of this license, visit [https://creativecommons.org/licenses/by-sa/4.0/legalcode](https://creativecommons.org/licenses/by-sa/4.0/legalcode).
+
+### Citation Information
+
+If you are using this dataset in your work, please cite this publication:
+
+A. Otegi, A. Agirre, J. A. Campos, A. Soroa, and E. Agirre. Conversational Question Answering in Low Resource Scenarios: A Dataset and Case Study for Basque. Proceedings of The 12th Language Resources and Evaluation Conference, pp. 429-435, European Language Resources Association. 2020.
```

Comment:
Can you put the citation in bibtex format here please ? This way researchers can just copy paste this into their papers